roadmap.md

```diff
@@ -1,60 +1,40 @@
 
-## Current Quarter
+## Timelines
 
-### CY2022Q1 / FY2022Q3
+### Near Term
+- Node token caching support with Redis 
+- Internal pipeline improvements for easier release cadence 
+- Deprecation of passport-azure-ad and replacement with a new lightweight token validator 
+- Updates required by the May 2021 United States Executive Order on cybersecurity: [Executive Order on Improving the Nation's Cybersecurity | The White House](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/) 
 
-| Library         | Deliverable                       | Status      | ETA       | Notes                                                        |
-| :-------------- | :-------------------------------- | :---------- | :-------- | :----------------------------------------------------------- |
-| MSAL Browser    | Telemetry interface               | In Progress | Mar 2022  | To provide support for customers that are interested in library performance |
-| MSAL Browser    | Security improvements             | In Progress | July 2022 | Improvements for Chrome and Edge on Windows to leverage existing device accounts |
-| MSAL Browser    | Resiliency improvements           | In Progress | Mar 2022  | Improvements around accessing the metadata endpoint to improve library reliability |
-| MSAL Node       | Distributed cache support         | In Progress | Mar 2022  | Improvements and samples to better support distributed caches like Redis in MSAL Node |
- 
+### Mid Term
+- Improvements on Node documentation & samples  
+- Ivy library distribution of MSAL Angular 
+- Angular 14 support 
+- Sustainable strategy to release adoption drivers like library dependency changes  
+  
 
-### CY2021Q4 / FY2022Q2
+### Long Term
+- IDP agnostic support  
+- Enhancing MSALGuard 
+- Reducing package size 
+- Increased support for the AAD Backup Auth Services  
+- Browsers' solutions to federated identity without 3p cookie access (FedCM, Storage Access API, First Party Sets, etc) 
 
-| Library         | Deliverable                       | Status      | ETA       | Notes                                                        |
-| :-------------- | :-------------------------------- | :---------- | :-------- | :----------------------------------------------------------- |
-| MSAL Browser    | General performance improvements  | In Progress | Dec 2021  | End to end review on token performance and proposals for optimizations. |
 
-## Past Quarters
+### Past
+This is not a comprehensive list of all completed features, but rather a way to note when something moved from near term to complete recently. You can view all past [Release Notes of MSAL.js](https://github.com/AzureAD/microsoft-authentication-library-for-js/discussions/categories/releases).
```

Comment:
Links to the individual changelogs for libs may be more useful than discussions given that they are searchable and most customers may only be interested in one library or the other